src/main/java/games/strategy/engine/lobby/server/ModeratorController.java

```diff
@@ -311,9 +323,14 @@ public String getInformationOn(final INode node) {
     builder.append("\r\nHost Name: ").append(node.getAddress().getHostName());
     builder.append("\r\nIP Address: ").append(node.getAddress().getHostAddress());
     builder.append("\r\nPort: ").append(node.getPort());
-    builder.append("\r\nHashed Mac: ").append(mac != null && mac.startsWith(games.strategy.util.MD5Crypt.MAGIC + "MH$")
-        ? mac.substring(6)
-        : mac + " (Invalid)");
+    builder.append("\r\nHashed Mac: ");
+    if (UNKNOWN_HASHED_MAC_ADDRESS.equals(mac)) {
+      builder.append("(Unknown)");
+    } else if (mac.startsWith(games.strategy.util.MD5Crypt.MAGIC + "MH$")) {
```

Comment:
I considered removing this check.  It seems to have been added to hide the constant `$1$MH$` prefix of all hashed MAC addresses from the mods.  I'm not sure if that's still necessary given how prevalent that prefix is throughout the rest of the UI (e.g. in an offline ban by MAC), as well as the DB that (at least some) mods presumably will view.

Leaving it in for now, but if there is agreement in displaying the complete hashed MAC address to the mods, I can remove it in a follow-up PR.

Reply:
That prefix was a constant pia for us it should just be auto added if possible just my 2 cents

Reply:
@ssoloff  i had to teach all to type it to make this clear

Reply:
@prastle Thanks for the background.  So it sounds like, if anything, we should attempt to hide the prefix in the rest of the UI rather than expose it here.  Noted.

Reply:
@ssoloff (Or just avoid hashing the mac altogether, because a MAC is not really sensitive information)